content/How to customize the display name of ActiveAdmin search filters- Juan Ferrari.md

```diff
@@ -0,0 +1,79 @@
+
+## Introduction
+
+At Rootstrap, my first assignment was to develop a Ruby on Rails application which uses ActiveAdmin (with ActiveAdmin Addons) as a framework to implement the user interface.
+
+I had never used ActiveAdmin before and I found myself reading a lot of documentation of the gem, in some cases, the documentation was not enough. For example, when I tried to display a custom name on an AJAX's search select filter at first it was a little tricky. So today I will show you how to do it, in the end, it's really simple!
+
+## Overview
+
+In this tutorial, you will learn how to customize the `display_name` of a `search_select` filter provided by ActiveAdmin Addons
+
+### Before start
+
+We have to have in our `Gemfile` the `activeadmin` gem and `activeadmin-addons` gem.
+
+In this example we are going to use a search select to filter people and search a person, our `person` model has two fields named `first_name` and `last_name`, and we want to display the full name of the person when we are searching it. The `full_name` is the concatenation of `first_name` and `last_name`.
```

Comment:
I'd remove the The on  
The `full_name` is the concatenation of `first_name` and `last_name`